server/utils/cloudFunctions/functions/test/algoliaTests/algoliaIntegrationTests.test.js

```diff
@@ -0,0 +1,242 @@
+/**
+ * Description: Integration tests for algolia cloud functions.
+ *              Can be flaky. If an error like `ObjectID` does not exist, 
```

Comment:
Is it the tests that are flaky or Algolia itself?  Don't submit flaky tests.  You will hate yourself for it later when you want to make a quick change to these tests and never know whether the failure is due to your new test or due to flakiness.

Plus in a production engineering environment flaky tests are a high-priority catch-and-kill task.  They drive engineers absolutely up the wall.

Reply:
Algolia is flaky, the tests are fine. To clarify the flakiness, I'm experiencing 1 sub test failing for every 10 runs due to algolia flakiness. I will update the JSdocs to de-emphasize this.